components/mitsubishi_itp/sensor/__init__.py

```diff
@@ -51,6 +84,14 @@
             accuracy_decimals=1,
             icon="mdi:sun-thermometer-outline",
         ),
+        CONF_RUNTIME: sensor.sensor_schema(
+            RuntimeSensor,
+            unit_of_measurement=UNIT_MINUTE,
+            device_class=DEVICE_CLASS_DURATION,
+            state_class=STATE_CLASS_TOTAL_INCREASING,
+            entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
+            accuracy_decimals=0,
+        ),
```

Comment:
Should this be hours instead? HA behaves better with hours than it does minutes, but minutes is closer to what the system knows.

Reply:
What's the minutes vs hours difference in Home Assistant? I'd be inclined toward reporting values more in line with the system and HA users can create overlay sensors if they want it compiled differently.

Reply:
At least on my end, minute shows `12345m` in Home Assistant while hours shows `123h 45m`. Catch with hours is we'd need two decimals of accuracy to properly show minutes. I don't know if there's a Home Assistant override to change units, might be though.

Reply:
I think minutes for now since that's what's being reported and if there's UI/display issues those can be addressed later.